integrations/slack/events/channel.go

```diff
@@ -80,6 +81,9 @@ func ChannelGroupHandler(l *zap.Logger, w http.ResponseWriter, body []byte, cb *
 		}
 	}
 
+	// Workaround for ENG-980: "is_moved: 0" should be "is_moved: false".
+	body = bytes.ReplaceAll(body, []byte(`"is_moved":0`), []byte(`"is_moved":false`))
```

Comment:
Why are we changing the event that comes from slack?

Reply:
That's the point of ENG-980: `is_moved` **should** be a boolean, but it's sent as a number, thereby breaking Go's JSON unmarshaling

Reply:
see previous comment - change approach to have more permissive JSON parsing